### ISSUE TITLE
Query regarding the Maintainers

### ISSUE THREAD

**Issue by Arna-Maity** [ComicoTeam/HavocOS_Official_Devices#1]
Hi @ComicoTeam, This list only includes those devices which are actively being maintained right?
Could you please also add the xda links and paypal links of the maintainers? Actually i was thinking of making a website for havoc os for which this database could be useful. If you don't mind, then can i PM you on telegram so that i could send you a video of my progress so far on the website. You can then share your views on the website.

**Comment by ComicoTeam** [comment]
Yes this list contains only the devices that are actively maintained right now. I will add PayPal and xda info. For website I need to talk with the creator of HavocOS since I didn't made it

**Comment by Arna-Maity** [comment]
I did talk with Anushek, he is one of the founders, he seemed to be interested.

**Comment by ComicoTeam** [comment]
He said to talk to him
